addon/-private/system/store/common.js

```diff
@@ -34,23 +33,12 @@ export function _objectIsAlive(object) {
   return !(get(object, 'isDestroyed') || get(object, 'isDestroying'));
 }
 
-let ASYNC_REQUEST_COUNT = 0;
-export function incrementRequestCount() {
-  ASYNC_REQUEST_COUNT++;
-}
-
-if (DEBUG) {
-  Ember.Test.registerWaiter(() => {
-    return ASYNC_REQUEST_COUNT === 0;
-  });
-}
-
 export function guardDestroyedStore(promise, store, label) {
   let wrapperPromise = resolve(promise, label).then(v => promise);
 
```

Comment:
can we put the incrementer here? If so, that would likely bit-rot related tester waiter imbalances.